dashboard/config/locales/scripts.en.yml

```diff
@@ -2295,7 +2295,13 @@ en:
               name: CSP Student Post-Course Survey
             Post-Course Survey:
               name: Post-Course Survey
-          lesson_groups: {}
+          lesson_groups:
+            cspSurvey:
+              display_name: Survey
+            csp1_1:
+              display_name: 'Chapter 1: Representing and Transmitting Information'
+            csp1_2:
+              display_name: 'Chapter 2: Inventing the Internet'
```

Comment:
why do we need a display_name key under the lesson group name/key? unless there are other translatable strings we plan to put under each one of these, it seems like it would be simpler to store these as follows:
```
          lesson_groups:
            cspSurvey: Survey
            csp1_1: 'Chapter 1: Representing and Transmitting Information'
            csp1_2: 'Chapter 2: Inventing the Internet'

```
apologies for not catching this earlier -- it was hard to spot before seeing the actual .yml file.

Reply:
If we go with this approach, is it easy to change later if we do find we want to add other strings?

Reply:
good question. I'm not sure. @Hamms , what will the process look like if we have to go from the flat format I'm proposing in my comment to the nested format shown the PR diff? Would we end up having to change the key `lesson_groups` to something else in the process, so that we could keep using the old translations while we wait for the i18n sync process to populate the values in the nested format?

Reply:
Well chapters in Curriculum Builder have descriptions and other fields we will likely want to translate when we move them over so it seems like we should keep display_name ?

Reply:
ok, that's fine. I suppose there's really no harm in doing it this way.